Sources/UIView+Sweet.swift

```diff
@@ -66,9 +66,12 @@ public extension UIView {
 
     /// Helper method to instantiate a nib file.
     ///
+    /// - Parameter nibName: The name of the nib to be loaded. If none is passed, it uses the class name. Defaults to nil.
     /// - Parameter bundle: The bundle where the nib is located, by default we'll use the main bundle.
+    ///
     /// - Returns: Returns an instance of the nib as a UIView.
-    class func instanceFromNib<T: UIView>(bundle: Bundle = .main) -> T {
-        return UINib(nibName: String(describing: T.self), bundle: bundle).instantiate(withOwner: nil, options: nil)[0] as! T
+    public class func instanceFromNib<T: UIView>(nibName: String? = nil, bundle: Bundle = .main) -> T {
```

Comment:
i think it would be better to return optional as instantiation from a nib works. We are never sure passed parameters are valid for nib loading and everything is safe. Thus i would return `T?` and get rid of force unwrap.